decidim-meetings/app/commands/decidim/meetings/admin/destroy_meeting.rb

```diff
@@ -37,6 +38,10 @@ def destroy_meeting
             meeting.destroy!
           end
         end
+
+        def proposals
+          @proposals ||= meeting.proposals
```

Comment:
allow Rails to memoize
```suggestion
          @proposals ||= meeting.proposals.load
```